### DISCUSSION TITLE
patrol and Navigator 2

### DISCUSSION THREAD

**Discussion by apkpotty** [leancodepl/patrol#627] [Q&A] [answered]
Is there anything specific required to be done to get Navigator 2 working with patrol?

"patrol drive" works till the point Navigator.pushNamed is called ...

**Comment by bartekpacia** [comment, marked as answer]
Hi @apkpotty 👋

> Is there anything specific required to be done to get Navigator 2 working with patrol?

No, it doesn't matter at all. At LeanCode all our apps use Navigator 2 and we're testing them with Patrol.

> "patrol drive" works till the point Navigator.pushNamed is called ...

That's probably a problem with your test. Please share the [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) and I'll try to help you.
